client/my-sites/earn/home.tsx

```diff
@@ -261,6 +279,8 @@ export default connect< ConnectedProps, {}, {} >(
 			siteId: site.ID,
 			selectedSiteSlug,
 			isFreePlan: ! isCurrentPlanPaid( state, site.ID ),
+			isJetpack: isJetpackSite( state, site.ID ),
+			isATSite: isSiteAutomatedTransfer( state, site.ID ),
```

Comment:
Not a biggie but would you be happy with `isAtomic` or `isAtomicSite`? It's already used like that all across Calypso.

Reply:
Ok, will make the change